backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -600,3 +610,8 @@ def _get_oidc_props(key_id: str) -> Tuple[Optional[str], Optional[str]]:
         raise ValueError("Missing OIDC configuration parameters.")
 
     return (oidc_keys[key_id], oidc_client_name)
+
+
+@cached(cache=TTLCache(maxsize=1, ttl=10))
+def _get_app_config() -> AppConfigurationModel:
```

Comment:
This feels like it should be a global util and not limited to the authorizer. I feel like very soon this will need to be used across a bunch of different lambdas based on all the upcoming tasks we have. Is there a reason this is a local function?

Reply:
That's a good point, I'll move it somewhere common and make it public

Reply:
Got an interesting error after moving it to `common_functions`

```
ImportError while loading conftest '/Users/odoug/workspace/MLSpace/mlspace/backend/test/conftest.py'.
test/conftest.py:23: in <module>
    from ml_space_lambda.data_access_objects.dataset import DatasetModel
src/ml_space_lambda/data_access_objects/dataset.py:25: in <module>
    from ml_space_lambda.data_access_objects.dynamo_data_store import DynamoDBObjectStore
src/ml_space_lambda/data_access_objects/dynamo_data_store.py:25: in <module>
    from ml_space_lambda.utils.common_functions import retry_config
src/ml_space_lambda/utils/common_functions.py:26: in <module>
    from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel
src/ml_space_lambda/data_access_objects/app_configuration.py:24: in <module>
    from ml_space_lambda.data_access_objects.dynamo_data_store import DynamoDBObjectStore
E   ImportError: cannot import name 'DynamoDBObjectStore' from partially initialized module 'ml_space_lambda.data_access_objects.dynamo_data_store' (most likely due to a circular import) (/Users/odoug/workspace/MLSpace/mlspace/backend/src/ml_space_lambda/data_access_objects/dynamo_data_store.py)
```

Looks like being in common functions is causing a circular dependency for some reason.

Reply:
Resolved it by creating a new `app_config_utils` file without any cross dependencies. Unit tests are running/succeeding again